README.md

```diff
@@ -16,11 +16,31 @@ Yet another chat-with-documents app, but supporting query over millions of files
 
 ### Overview
 
-ChatData is a LLM-based chat app, which brings unmatched efficiency and accuracy to your document interactions. Utilizing the Retrieval Augmented Generation (RAG) framework, ChatData takes millions of wikipedia pages and arxiv papers as the external knowledge base with MyScale handling all the data hosting jobs. Now say goodbye to cumbersome keyword searches and hello to a seamless experience with powerful metadata filters and advanced vector search capabilities. 
+ChatData is a robust chat-with-documents application designed to extract information and provide answers by querying the MyScalefree knowledge base or your uploaded documents.
 
-With ChatData, you can effortlessly navigate through vast amounts of data, effortlessly accessing precisely what you need. Whether you're a researcher, a student, or a knowledge enthusiast, ChatData empowers you to explore academic papers and research documents like never before. Unlock the true potential of information retrieval with ChatData and discover a world of knowledge at your fingertips.
+Powered by the Retrieval Augmented Generation (RAG) framework, ChatData leverages millions of Wikipedia pages and arXiv papers as its external knowledge base, with MyScale managing all data hosting tasks. Simply input your questions in natural language, and ChatData takes care of generating SQL, querying the data, and presenting the results.
 
-![](/assets/rag-enabled-chatdata.gif)
+Enhancing your chat experience, ChatData introduces three key features. Let's delve into each of them in detail.
+
+#### Feature 1: Retriever Type
+
+**Self-querying retriever:**
```

Comment:
please help write a paragraph to introduce self-querying retriever, thanks @mpskex